inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -149,11 +152,11 @@ std::pair<bool, std::string> compare_functions(
                 auto const2 = ngraph::as_type_ptr<Constant>(node2->get_input_node_shared_ptr(i));
 
                 const auto equal = [](const Constant &c1, const Constant &c2) {
-                    const auto equal_float_str = [](const std::string &s1, const std::string s2) {
-                        return std::abs(std::stof(s1) - std::stof(s2)) < 0.001;
+                    const auto equal_float_str = [](const double &s1, const double & s2) {
```

Comment:
`std::string`s are not compared now. Name is misleading.

Reply:
done.